### ISSUE TITLE
Sustainable Web Design Score in CO2.js

### ISSUE THREAD

**Issue by benjaminrancourt** [thegreenwebfoundation/co2.js#219]
_If this isn't the most appropriate place to discuss this, forgive me. I preferred to do it here rather than attempt to have a discussion on a closed Merge Request._

In #207, the calculation of the score according to the Sustainable Web Design methodology was introduced in CO2.js, but I wonder about its implementation. In my understanding of the methodology, the score is based on the data transfer size, but the current implementation in CO2.js is, instead, based on the carbon emissions that have been calculated...

https://github.com/thegreenwebfoundation/co2.js/blob/8f9539973fad0555ee37d20a609922b412d1fb8b/src/helpers/index.js#L258-L298

What is the reasoning for making such a change?

Thanks in advance for the explanations! 🤗

**Comment by fershad** [comment]
@benjaminrancourt Yes, data transfer is a key input for the Sustainable Web Design Model formula, however it's not the sole basis for estimating carbon emissions using that model. There are other factors (green hosting, caching, etc) that can also be applied. As a result, basing the rating system off the result of the carbon estimate allows these factors to also be considered, rather than it just being a straight 1:1 relationship with data transfer.

**Comment by fershad** [comment]
Closing this, as no follow up since September.
